### PR TITLE
subl: add page

### PR DESCRIPTION
**Pull request by tpv-ebben**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/subl.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Sublime Text, a sophisticated text editor for code, markup and prose.
+
+- Open a file or folder:
+
+`subl {{path/to/file_or_folder}}`
+
+- Open a file or folder in a new window:
+
+`subl -n {{path/to/file_or_folder}}`
```
Comment (sbrl) [inline pages/common/subl.md]: Is there a long form of this argument at all? We do generally prefer them if they are available. Same for the other examples below.

File: pages/common/subl.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Add a folder to existing window:
+
+`subl -a {{path/to/folder}}`
+
+- Open a Project:
+
+`subl --project {{path/to/project/file}}`
```
Comment (sbrl) [inline pages/common/subl.md]: Do project files have a specific extension? Perhaps we could make it `{{path/to/project/file.ext}}`, where `ext` is the preferred file extension for sublime text project files.

File: pages/common/subl.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# subl
+
+> Sublime Text, a sophisticated text editor for code, markup and prose.
```
Comment (sbrl) [inline pages/common/subl.md]: Teensy grammatical correction: `markup and prose` => `markup, and prose`. ([ref](https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma))

File: pages/common/subl.md (hunk 1 above)
Comment (agnivade) [reply pages/common/subl.md]: Seems fine to me. Since there is a word (new) whose first letter matches the argument. Same for -a -> Add.

Comment (sbrl) [reply pages/common/subl.md]: True, @agnivade

File: pages/common/subl.md (hunk 2 above)
Comment (pxgamer) [reply pages/common/subl.md]: According to the docs, it looks like there are 2 files extensions.
`.sublime-project` - the project definition
`.sublime-workspace` - user specific data

I'm assuming that this command would use the first one.

Comment (sbrl) [reply pages/common/subl.md]: Yeah!

**Pull request closed without merging**